src/main/java/eu/execom/pomodoroTeam/controllers/TeamController.java

```diff
@@ -0,0 +1,126 @@
+package eu.execom.pomodoroTeam.controllers;
+
+import eu.execom.pomodoroTeam.controllers.util.RESTError;
+import eu.execom.pomodoroTeam.entities.UserEntity;
+import eu.execom.pomodoroTeam.services.TeamService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import eu.execom.pomodoroTeam.entities.TeamEntity;
+import eu.execom.pomodoroTeam.entities.dto.TeamDto;
+import eu.execom.pomodoroTeam.repositories.TeamRepository;
+import eu.execom.pomodoroTeam.repositories.UserRepository;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "/team")
+public class TeamController {
+
+    private TeamRepository teamRepository;
+    private UserRepository userRepository;
+    private TeamService teamService;
+
+    @Autowired
+    public TeamController(TeamRepository teamRepository, UserRepository userRepository, TeamService teamService) {
+        this.teamRepository = teamRepository;
+        this.userRepository = userRepository;
+        this.teamService = teamService;
+    }
+
+    /**
+     * creat new team
+     *
+     * @param teamDto
+     * @return
+     */
+    @RequestMapping(method = RequestMethod.POST)
+    public ResponseEntity<?> createNewTeam(@RequestBody TeamDto teamDto) {
+        TeamEntity team = new TeamEntity();
+        team.setName(teamDto.getName());
+        teamRepository.save(team);
+        return new ResponseEntity<>(team, HttpStatus.OK);
+    }
+
+    /**
+     * list of all teams
+     *
+     * @return
+     */
+
+    @RequestMapping(method = RequestMethod.GET, value = "/getAllTeams")
```

Comment:
Please do not leave empty space in between JavaDoc and method headers or annotations above them.